### PR TITLE
update dev approval ux

### PR DESCRIPTION
**Pull request by mgaoVA**

### CODE REVIEW

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=4992809&bid=16480344)

File: LEAF_Request_Portal/utils/db_upgrade/Update_RMC_DB_2020011000-20200207.sql [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
...nput class=\"icheck_rob\" type=\"checkbox\" id=\"rob_acceptance\" value=\"Accepted terms and rules of behavior\" style=\"width: 32px\" /> I understand and accept. <span id=\"rob_required\" style=\"margin-left: 8px; color: red\">*&nbsp; Required</span>\r\n</div>', `htmlPrint` = '<style>\r\n    #devconsole_description > p, #devconsole_description > ul > li {\r\n        font-size: 16px;\r\n    }\r\n</style>\r\n<div id=\"devconsole_description\">\r\n<p><b>This is a request to access the LEAF Developer Console.</b></p>\r\n<p>Approved individuals will gain the ability to modify LEAF\'s user interface using HTML, CSS, and JavaScript technologies.</p>\r\n<p>By approving this request:\r\n<ul>\r\n    <li>I validate that the person requesting access as the technical ability to work with HTML, CSS, and JavaScript source code.</li>\r\n    <li>I recognize that source code customizations and their maintenance are the responsibility of the office seeking to make customizations. In the event that the responsible office becomes unable to support maintenance, customizations may be easily removed to restore standard functionality.</li>\r\n</ul>\r\n</p>\r\n</div>' WHERE `indicators`.`indicatorID` = -5;
```
Comment (dastan-rahimi) [inline LEAF_Request_Portal/utils/db_upgrade/Update_RMC_DB_2020011000-20200207.sql]: Fortify is flagging this line.

Comment (thomasnguyen95) [reply LEAF_Request_Portal/utils/db_upgrade/Update_RMC_DB_2020011000-20200207.sql]: This file should be renamed to match the convention unless conventions are changing again.

**Comment by dastan-rahimi** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 8
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/VALEAF/LEAF/pullRequest?prid=4992809&bid=16480344)

**Pull request merged**